packages/material-ui/src/Slide/Slide.test.js

```diff
@@ -39,19 +30,19 @@ describe('<Slide />', () => {
       inheritComponent: 'Transition',
       mount,
       refInstanceof: React.Component,
-      skip: ['componentProp'],
+      skip: ['componentProp', 'refForwarding'],
```

Comment:
Good idea.

Reply:
I guess this is a good compromise, I'm not sure why someone would use the ref forwarding feature on Transitions manually. We should probably allow it everything since parent components might need the feature.